### PR TITLE
Milestone 0.3.0 issues

### PR DESCRIPTION
**Pull request by slkennedy**
Fixes a number of Bug, reactors a button and cards, and adds to button documentation. Refer to Issues for details.

#10 - add detail to Button docs
#58 - mixin for button modifier styling
#59 - remove/fix button[href] styling 
#60 - refactor card to prevent weird growing / implement better card-grid
#61 - skip-nav not displaying on focus
#62 - primary-nav links need a focus style

### CODE REVIEW

**Comment by slkennedy** [comment]
I just pushed updates to address the issue with link focus and active styling.

File: components/banner/styles/_heading.scss [hunk 1]
```diff
@@ -1,3 +1,3 @@
 &__heading {
-    @include set-text-style($text-style-heading-1);
```
[the remark this answers is not in the document]
Comment (Keale2) [reply components/banner/styles/_heading.scss]: We prescribe markup for this component, so I think it's fine to leave this style out. If a developer uses custom markup for this component, they are responsible for adjusting styles. We can revisit this if we run into issues.

[the remark this answers is not in the document]
Comment (chuckhendo) [reply components/banner/styles/_heading.scss]: I think having global styles is a necessity, however, there will be times they will need to be disabled (such as with email and tables). I feel like we should leave it as a safety net, but I don't feel strongly about that

**Pull request merged**